src/main/java/teammates/logic/api/EmailGenerator.java

```diff
@@ -456,23 +456,40 @@ private EmailWrapper generateFeedbackSessionEmailBaseForInstructors(
 
         List<EmailWrapper> emails = new ArrayList<EmailWrapper>();
 
+        String instructorFragmentArgumentInstructors = "<p>The email below has been sent to students of course: "
+                + SanitizationHelper.sanitizeForHtml(course.getId()) + ", "
+                + SanitizationHelper.sanitizeForHtml(course.getName())
+                + ".<br>" + Const.EOL + "<br>" + Const.EOL
+                + "=== Email message as seen by the students ===</p>" + Const.EOL;
```

Comment:
This fragment is similar to the one used in `generateFeedbackSessionEmailBaseForInstructors` so it would be good if you can extract a method to generate this fragment or use some sort of html template to generate this string to reduce the repetition.